src/tools/wdump/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+add_executable(wdump WIN32)
```

Comment:
Shouldn't you add this executable to the list of artifacts to be uploaded during ci compile

Reply:
Fixed.